hibernate-reactive-core/src/test/java/org/hibernate/reactive/IdentityGeneratorTypeForCockroachDBTest.java

```diff
@@ -78,8 +78,8 @@ public void longIdentityType(TestContext context) {
 
 	@Test
 	public void integerIdentityType(TestContext context) {
-		thrown.expectMessage( "too big" );
-		thrown.expectMessage( "Integer" );
+//		thrown.expectMessage( "too big" );
+//		thrown.expectMessage( "Integer" );
```

Comment:
@DavideD what was the point of these tests? What were they trying to prove?

Reply:
I think cockroachDB generates ids that are too big to be saved in an Integer or Short.
So the test is really about checking that a meaningful exception is thrown.

We should probably add a comment about it

Reply:
Why would you uncomment this?

Reply:
I guess you've also fixed this "issue" for CockroachDB

Reply:
You can remove this class and update `IndentityGeneratorTypeTest`  so that the tests are the same for every db

Reply:
See https://github.com/hibernate/hibernate-reactive/blob/41f34afb2e46f2502899a5387229a939185d338f/hibernate-reactive-core/src/test/java/org/hibernate/reactive/IdentityGeneratorTypeTest.java#L37

Reply:
> Why would you uncomment this?

Well because with this change the tests stopped failing.

Reply:
> Well because with this change the tests stopped failing.

Yes, I've sent other messages after this.

Reply:
Yeah but what I'm asking is: where those exceptions *necessary* for some reason. Have I broken things by "fixing" it?

Reply:
This is a mapping a user might try (or have tried, I can't remember) and it makes sure that the right exception get thrown.

Now you have inadvertently changed the behaviour when using CockroachDB.
I guess in this case you have changed it for the better, so there shouldn't be any negative consequence.

I don't think anybody is using `integer` or `short` on purpose with the identity type so that they can see the exceptions. At least we know we can remove some duplicated test classes.

Reply:
OK, fine, I will remove it then.

Reply:
Wait... now that I think about it, you have removed a validation check. I guess the identifier with CockroachDB will become pretty long pretty soon and we won't be able to set it to a `short`. What will happen in that case?

Reply:
Well, `throw LOG.generatedIdentifierTooBigForTheField()`, I suppose.

Reply:
Wait, suddenly the exception happens for me again.

Reply:
I dunno, something a bit fishy there, but anyway the tests are doing what they're supposed to do again now.